### DISCUSSION TITLE
Alternative to --download-url-prefix?

### DISCUSSION THREAD

**Discussion by coryl** [sparkle-project/Sparkle#2490] [Q&A] [answered]
I'm looking for something similar to this argument on `generate_appcast`, something to help customize the URL of the update.

Basically my file URL doesn't conform to the structure used by the script. (I'm uploading to Firebase storage in my build process, so my hosting URLs look something like: `https://firebasestorage.googleapis.com/v0/b/myapp-343c6.appspot.com/o/builds/updates/myapp1.1.3.zip?alt=media/ `

Before I write some custom python or shell to open up appcast.xml and replace the `<enclosure url="">` values, was wondering if anyone had a similar issue and what they did. Thanks

**Comment by zorgiepoo** [comment]
`--download-url-prefix` seems like the closest thing `generate_appcast` has I think. Does that option not work because you need to add `?alt=media/ ` at the end?

**Reply by coryl** [reply, marked as answer]
Yes, its okay though we just made the entire bucket public, resolving our url structure issue.
